examples/WorkflowSubmission.java

```diff
@@ -0,0 +1,47 @@
+import com.indico.IndicoClient;
+import com.indico.IndicoConfig;
+import com.indico.entity.Workflow;
+import com.indico.jobs.Job;
+import com.indico.mutation.WorkflowSubmission;
+import com.indico.query.ListWorkflowsForDatasetQuery;
+import com.indico.storage.Blob;
+import com.indico.storage.RetrieveBlob;
+import com.indico.type.JobStatus;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import org.json.JSONObject;
+
+public class WorkflowSubmission {
+
+    public static void main(String args[]) throws IOException {
+        IndicoConfig config = new IndicoConfig.Builder()
+                .host("dev.indico.io")
```

Comment:
lets change this to app.indico.io dev not technically public

Reply:
Fixed